proto/buf/alpha/registry/v1alpha1/plugin_curation.proto

```diff
@@ -27,6 +27,18 @@ enum PluginRegistryType {
   PLUGIN_REGISTRY_TYPE_NPM = 2;
 }
 
+// PluginLang is used to specify the languages a plugin supports.
+enum PluginLang {
```

Comment:
We could name this `PluginLanguage` instead of this abbreviated `Lang` format. And use `language` in other places instead of `lang`, thoughts?

Reply:
Let's just use `Language/language` throughout for consistency.